tests/test_dsp.py

```diff
@@ -218,6 +218,10 @@ def test_regularized_spectrum_inversion_assertions(impulse):
             TypeError, match="The normalized parameter"):
         dsp.regularized_spectrum_inversion(impulse, [200, 10e3], normalized=1)
 
+    with pytest.raises(
+            ValueError, match="Frequency range must be provided"):
+        dsp.regularized_spectrum_inversion(impulse)
+
```

Comment:
```suggestion
```

the logic of the old funtionality should not change, the only possible change is from ``freq_range`` to ``frequency_range``